lib/helpers.ts

```diff
@@ -1,7 +1,15 @@
 export const urlEncodeBase64 = (signature: string): string => {
-  signature = signature.replace(/(=+)$/g, '');
-  signature = signature.replace(/\//g, '_');
-  signature = signature.replace(/\+/g, '-');
+  signature = signature.replace(/(=+)$/g, "");
+  signature = signature.replace(/\//g, "_");
+  signature = signature.replace(/\+/g, "-");
 
   return signature;
 };
+
+export const urlSafeBase64ToBase64 = (signature) => {
+  if (isUrlSafeBase64(signature))
+    return signature.replace(/-/g, "+").replace(/_/g, "/");
+};
```

Comment:
Could you please apply this style?
```suggestion
  if (isUrlSafeBase64(signature)) {
    return signature.replace(/-/g, '+').replace(/_/g, '/');
  }
};
```

Also, what happens if the if statement returns false? Do you not need to return the input signature in that case?

```
export const urlSafeBase64ToBase64 = (signature) => {
  if (isUrlSafeBase64(signature)) {
    return signature.replace(/-/g, '+').replace(/_/g, '/');
  }

  return signature;
}

Reply:
Well😄, I'll resubmit it as soon as I can.

Reply:
```ts
export const urlSafeBase64ToBase64 = (signature) => {
  if (isUrlSafeBase64(signature)) {
    return signature.replace(/-/g, '+').replace(/_/g, '/');
  }

  return signature;
}
```

In the given code, if the if condition evaluates to false, the function proceeds to the return signature; statement after the if block.

Here’s a breakdown:
	1.	The if statement checks if the input signature is in a URL-safe Base64 format using the function isUrlSafeBase64.
	2.	If isUrlSafeBase64(signature) returns true, the function modifies the signature by replacing - with + and _ with / and then returns the modified value.
	3.	If isUrlSafeBase64(signature) returns false, the function skips the modification and directly returns the original input signature.

So yes, the function always returns the signature, either in its modified form (if it’s URL-safe) or in its original form (if it’s not URL-safe). This ensures that the function has a valid return value regardless of the condition.